# arm64 builds for nf-core pipelines

Conda packages found from source code and independently built using Wave, without a version number.
Checks are only for build success, not for functionality.

> [!NOTE]
> Table last updated 2025-04-30

| Pipeline                  | Success % | Succeeded | Failed |
| ------------------------- | --------- | --------- | ------ |
| rnaseq                    | 100.0%    | 52        | 0      |
| sarek                     | 100.0%    | 41        | 0      |
| mag                       | 88.9%     | 40        | 5      |
| scrnaseq                  | 100.0%    | 20        | 0      |
| chipseq                   | 100.0%    | 25        | 0      |
| atacseq                   | 100.0%    | 23        | 0      |
| ampliseq                  | 100.0%    | 16        | 0      |
| nanoseq                   | 100.0%    | 20        | 0      |
| fetchngs                  | 100.0%    | 9         | 0      |
| eager                     | 95.6%     | 43        | 2      |
| rnafusion                 | 89.2%     | 33        | 4      |
| methylseq                 | 100.0%    | 19        | 0      |
| taxprofiler               | 100.0%    | 38        | 0      |
| viralrecon                | 97.5%     | 39        | 1      |
| hic                       | 100.0%    | 26        | 0      |
| raredisease               | 96.2%     | 51        | 2      |
| cutandrun                 | 100.0%    | 26        | 0      |
| pangenome                 | 70.0%     | 7         | 3      |
| smrnaseq                  | 100.0%    | 35        | 0      |
| funcscan                  | 78.8%     | 26        | 7      |
| differentialabundance     | 96.3%     | 26        | 1      |
| hlatyping                 | 100.0%    | 13        | 0      |
| bacass                    | 80.6%     | 25        | 6      |
| proteinfold               | 100.0%    | 12        | 0      |
| airrflow                  | 85.7%     | 6         | 1      |
| oncoanalyser              | 100.0%    | 30        | 0      |
| bactmap                   | 97.1%     | 33        | 1      |
| spatialvi                 | 100.0%    | 14        | 0      |
| rnasplice                 | 100.0%    | 18        | 0      |
| demultiplex               | 100.0%    | 18        | 0      |
| circrna                   | 96.3%     | 52        | 2      |
| scdownstream              | 100.0%    | 41        | 0      |
| epitopeprediction         | 85.7%     | 18        | 3      |
| rnavar                    | 96.0%     | 24        | 1      |
| crisprseq                 | 94.7%     | 18        | 1      |
| proteomicslfq             | 94.7%     | 18        | 1      |
| mhcquant                  | 77.8%     | 7         | 2      |
| lncpipe                   | 100.0%    | 33        | 0      |
| isoseq                    | 82.4%     | 14        | 3      |
| circdna                   | 100.0%    | 12        | 0      |
| readsimulator             | 100.0%    | 14        | 0      |
| imcyto                    | 100.0%    | 1         | 0      |
| hgtseq                    | 100.0%    | 16        | 0      |
| multiplesequencealign     | 100.0%    | 24        | 0      |
| metatdenovo               | 94.1%     | 32        | 2      |
| gwas                      | 100.0%    | 6         | 0      |
| deepmodeloptim            | 100.0%    | 5         | 0      |
| genomeannotator           | 100.0%    | 3         | 0      |
| bamtofastq                | 100.0%    | 5         | 0      |
| scnanoseq                 | 94.1%     | 16        | 1      |
| clipseq                   | 90.9%     | 20        | 2      |
| fastquorum                | 100.0%    | 6         | 0      |
| dualrnaseq                | 100.0%    | 11        | 0      |
| nascent                   | 97.6%     | 41        | 1      |
| phaseimpute               | 90.5%     | 19        | 2      |
| viralintegration          | 100.0%    | 6         | 0      |
| metaboigniter             | 100.0%    | 2         | 0      |
| genomeassembler           | 86.1%     | 31        | 5      |
| diaproteomics             | 91.7%     | 33        | 3      |
| pathogensurveillance      | 88.6%     | 39        | 5      |
| detaxizer                 | 100.0%    | 8         | 0      |
| variantbenchmarking       | 77.8%     | 21        | 6      |
| nanostring                | 100.0%    | 19        | 0      |
| riboseq                   | 100.0%    | 33        | 0      |
| metapep                   | 100.0%    | 3         | 0      |
| cageseq                   | 100.0%    | 13        | 0      |
| phageannotator            | 74.1%     | 20        | 7      |
| omicsgenetraitassociation | 100.0%    | 6         | 0      |
| molkart                   | 66.7%     | 4         | 2      |
| mnaseseq                  | 95.0%     | 19        | 1      |
| denovotranscript          | 84.6%     | 11        | 2      |
| tfactivity                | 70.0%     | 14        | 6      |
| reportho                  | 100.0%    | 5         | 0      |
| coproid                   | 92.6%     | 25        | 2      |
| variantcatalogue          | 92.9%     | 13        | 1      |
| pixelator                 | 85.7%     | 6         | 1      |
| mcmicro                   | 50.0%     | 1         | 1      |
| datasync                  | 100.0%    | 1         | 0      |
| createpanelrefs           | 100.0%    | 5         | 0      |
| tbanalyzer                | 94.1%     | 16        | 1      |
| slamseq                   | 100.0%    | 11        | 0      |
| seqinspector              | 100.0%    | 7         | 0      |
| phyloplace                | 100.0%    | 14        | 0      |
| hicar                     | 100.0%    | 24        | 0      |
| createtaxdb               | 100.0%    | 22        | 0      |
| radseq                    | 100.0%    | 18        | 0      |
| pairgenomealign           | 100.0%    | 6         | 0      |
| meerpipe                  | 100.0%    | 2         | 0      |
| marsseq                   | 95.5%     | 21        | 1      |
| rnadnavar                 | 92.5%     | 37        | 3      |
| rangeland                 | 100.0%    | 4         | 0      |
| pgdb                      | 100.0%    | 9         | 0      |
| genomeqc                  | 87.5%     | 14        | 2      |
| callingcards              | 100.0%    | 21        | 0      |
| methylarray               | 85.2%     | 23        | 4      |
| magmap                    | 100.0%    | 19        | 0      |
| genomeskim                | 100.0%    | 2         | 0      |
| demo                      | 100.0%    | 3         | 0      |
| troughgraph               | 100.0%    | 2         | 0      |
| sammyseq                  | 100.0%    | 23        | 0      |
| fastqrepair               | 100.0%    | 5         | 0      |
